S3/English/ASS_30.ass

```diff
@@ -0,0 +1,109 @@
+Dialogue: 0,0:01:45.52,0:01:47.47,Default,,0,0,0,,I will send a bill of damaged stuff to your email
+Dialogue: 0,0:01:47.77,0:01:48.88,Default,,0,0,0,,Anyway before you quit drinking
+Dialogue: 0,0:01:49.01,0:01:49.95,Default,,0,0,0,,Don't come to 1031 again
+Dialogue: 0,0:01:52.51,0:01:52.98,Default,,0,0,0,,What
+Dialogue: 0,0:01:53.16,0:01:54.50,Default,,0,0,0,,I broke all of that?!
+Dialogue: 0,0:01:58.65,0:01:59.25,Default,,0,0,0,,Above all
+Dialogue: 0,0:02:00.14,0:02:01.70,Default,,0,0,0,,Lily will live with you these days
+Dialogue: 0,0:02:02.27,0:02:03.02,Default,,0,0,0,,As a supervisor
+Dialogue: 0,0:02:03.22,0:02:04.61,Default,,0,0,0,,Hello, Momo
+Dialogue: 0,0:02:05.84,0:02:07.14,Default,,0,0,0,,So b
+Dialogue: 0,0:02:10.65,0:02:12.64,Default,,0,0,0,,Lily should be asleep now
+Dialogue: 0,0:02:13.28,0:02:14.22,Default,,0,0,0,,Now
+Dialogue: 0,0:02:15.38,0:02:16.30,Default,,0,0,0,,Lily you underestimated me so much
+Dialogue: 0,0:02:16.30,0:02:17.37,Default,,0,0,0,,Don't drink alcohol
+Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
+Dialogue: 0,0:02:20.80,0:02:22.10,Default,,0,0,0,,Don't drink alcohol
+Dialogue: 0,0:02:22.10,0:02:23.00,Default,,0,0,0,,Lily is watching you
+Dialogue: 0,0:02:25.07,0:02:25.97,Default,,0,0,0,,Don't drink alcohol
+Dialogue: 0,0:02:26.20,0:02:27.50,Default,,0,0,0,,Lily is watching you
+Dialogue: 0,0:02:28.15,0:02:29.63,Default,,0,0,0,,Lily is watching you
+Dialogue: 0,0:02:29.68,0:02:30.79,Default,,0,0,0,,Lily is watching you
+Dialogue: 0,0:02:30.90,0:02:32.12,Default,,0,0,0,,Lily is watching you
+Dialogue: 0,0:02:32.81,0:02:35.04,Default,,0,0,0,,How did you find me
+Dialogue: 0,0:02:37.44,0:02:38.40,Default,,0,0,0,,Lily, Momo
+Dialogue: 0,0:02:38.54,0:02:39.65,Default,,0,0,0,,I'm here with condolences
+Dialogue: 0,0:02:41.98,0:02:42.93,Default,,0,0,0,,Is Momo ok
+Dialogue: 0,0:02:43.21,0:02:44.84,Default,,0,0,0,,Hasn't drink any alcohol for two days
+Dialogue: 0,0:02:46.06,0:02:47.48,Default,,0,0,0,,I'll put juice and desserts here
+Dialogue: 0,0:02:49.30,0:02:50.03,Default,,0,0,0,,Who is this
+Dialogue: 0,0:02:51.31,0:02:52.23,Default,,0,0,0,,Look familiar
+Dialogue: 0,0:02:53.18,0:02:54.98,Default,,0,0,0,,That's the photo of me when I was an idol
+Dialogue: 0,0:02:55.18,0:02:55.86,Default,,0,0,0,,Idol?
+Dialogue: 0,0:02:57.71,0:02:58.43,Default,,0,0,0,,Hello everyone
+Dialogue: 0,0:02:58.64,0:03:00.59,Default,,0,0,0,,We are
+Dialogue: 0,0:03:03.16,0:03:04.46,Default,,0,0,0,,Then why did you give up
+Dialogue: 0,0:03:05.12,0:03:06.46,Default,,0,0,0,,It's too much stress to be an idol
+Dialogue: 0,0:03:07.05,0:03:08.91,Default,,0,0,0,,But it's mainly my habit is difficult to change
+Dialogue: 0,0:03:09.34,0:03:11.28,Default,,0,0,0,,And I was photographed on the headlines
+Dialogue: 0,0:03:14.64,0:03:15.55,Default,,0,0,0,,How did they recognize you
+Dialogue: 0,0:03:15.55,0:03:16.79,Default,,0,0,0,,The reporter is so powerful
```

Comment:
Need a better tl

Reply:
"The reporter is so good"? (maybe "talented")

Reply:
Perhaps "The reporter has a really good eye" ?